.circleci/config.yml

```diff
@@ -486,14 +486,14 @@ jobs:
           command: |
             if [ ! -f BUILDS ]; then exit 0; fi
             cat > /etc/apt/sources.list \<< EOF
-            deb [arch=amd64] http://archive.ubuntu.com/ubuntu focal main restricted universe multiverse
-            deb [arch=amd64] http://security.ubuntu.com/ubuntu focal-updates main restricted universe multiverse
-            deb [arch=amd64] http://security.ubuntu.com/ubuntu focal-security main restricted universe multiverse
-            deb [arch=amd64] http://archive.ubuntu.com/ubuntu focal-backports main restricted universe multiverse
-            deb [arch=arm64] http://ports.ubuntu.com/ focal main restricted universe multiverse
-            deb [arch=arm64] http://ports.ubuntu.com/ focal-updates main restricted universe multiverse
-            deb [arch=arm64] http://ports.ubuntu.com/ focal-security main restricted universe multiverse
-            deb [arch=arm64] http://ports.ubuntu.com/ focal-backports main restricted universe multiverse
+            deb [arch=amd64] http://archive.ubuntu.com/ubuntu jammy main restricted universe multiverse
```

Comment:
I see some divergence in the dependencies, e.g. `gcc-9`. Should we fix those?
https://github.com/cilium/image-tools/blob/ff22ba3bff1010f4a2dd76ede789663c3beaf8d2/images/compilers/install-deps.sh